### ISSUE TITLE
Investigate atmosphere cometd

### ISSUE THREAD

**Issue by uklance** [uklance/tapestry-cometd#28]
Atmosphere claims to support all servlet containers and has a CometD servlet using the atmosphere framework.
https://github.com/Atmosphere/atmosphere/wiki/Getting-started-with-Cometd
http://atmosphere.github.com/atmosphere/apidocs/org/atmosphere/cometd/CometdServlet.html
http://atmosphere-framework.2306103.n4.nabble.com/Using-CometD-with-Atmosphere-td4650724.html

**Comment by jfarcand** [comment]
Salut, nay reason to have stopped using Atmosphere? Please let me know if you face any issue, I will be happy to fix them and help with the integration. Thanks

**Comment by uklance** [comment]
Hi jfarcand, I was actually trying to find a non-atmosphere solution to get a handle on the current HttpServletRequest. Turns out that this is not available when using a WebSocket. Since I only actually needed the HttpSession (and not the request) I found a solution [here](https://github.com/uklance/tapestry-cometd-demo/blob/master/src/main/java/org/lazan/t5/cometddemo/services/internal/ChatSubscriptionListener.java#L29).

Looks like I forgot to switch my pom.xml back to use atmosphete-cometd after solving my problem. I'll make sure I switch back before releasing the next version.

How very observant of you... github is the new big-brother ;)
